external/radiation/python/radlw/radlw_main.py

```diff
@@ -1726,14 +1723,6 @@ def lwrad(
         if verbose:
             print("Beginning lwrad . . .")
             print(" ")
-
-        if self.isubclw == 1:
```

Comment:
Similar to Noah's comment about `effr`, I would add raising an error if either `isubclw` or `isubcsw` isn't 2. Those two seem like the more commonly-changed flags that weren't implemented/validated in the port, as far as I know.

Reply:
Added raise error for when effr_in is set to True. 

isubclw and isubcsw are called at initialization so I added rise error for when they are set  to 2.